prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -102,7 +102,8 @@ spec:
           type: spc_t
       volumes:
         - name: weavedb
-          emptyDir: {}
+          hostPath:
+            path: /var/lib/weave-net
```

Comment:
Just let me know the final name (I'm fine with `/var/lib/weave/net`), and I will change it in the plugin-v2.